### DISCUSSION TITLE
[V6] Access database using Lucid Models inside a Ace command

### DISCUSSION THREAD

**Discussion by moabsi** [adonisjs/core#4655] [Help] [answered]
i am having issue with Models in command in Adonis V6
i am trying to use models in ace command for creating data like seeder but with data coming from an API i tried to import the Model Like this
const { default: Actions } = await import('#models/actions')

but still no luck with fixing it

**Comment by RomainLanz** [comment, marked as answer]
Hey @moabsi! 👋🏻 

You have to start the application inside your command.

📚 https://docs.adonisjs.com/guides/ace/creating-commands#command-metadata

**Reply by moabsi** [reply]
Hi @RomainLanz this absolutely fixed the problem seamlessly 

thanks for your quick reply
